ppr-api/src/ppr_api/models/client_code.py

```diff
@@ -83,7 +83,7 @@ def find_by_code(cls, code: str = None):
         return party
 
     @classmethod
-    def find_by_head_office_code(cls, head_office_id: str = None):
+    def find_by_head_office_code(cls, head_office_id: str):
```

Comment:
do we still need this? keep just in case?

Reply:
No harm in keeping.